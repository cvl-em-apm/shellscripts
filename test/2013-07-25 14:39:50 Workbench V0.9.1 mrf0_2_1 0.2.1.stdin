Logging started ...

2013-07-25 14:39:50+1000 > 

Logging stopped at 2013-07-25 16:07:57+1000

