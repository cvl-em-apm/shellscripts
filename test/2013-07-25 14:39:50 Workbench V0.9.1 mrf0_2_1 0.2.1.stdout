
Enter Data Filename: 
Error: data file  not found